### PR TITLE
[ADD] Project Tag Module

### PR DESCRIPTION
**Pull request by patrickrwilson**
Adds the Project Tag module.

Original Discussion #500

### CODE REVIEW

File: project_tag/__manifest__.py
Comment (dreispt) [inline project_tag/__manifest__.py]: Deprecated, please remove.

File: project_tag/README.rst
Comment (dreispt) [inline project_tag/README.rst:80]: I think you need to add a "readme" subdirectory with the several RST content fragments. See here: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme
This will then automatically generate (and overwrite) the top README.rst.

**Review by dreispt — COMMENTED** [review]
Mostly LGTM

File: project_tag/README.rst
Comment (patrickrwilson) [reply project_tag/README.rst:80]: My fault, I somehow got my readme files mixed up and I need to go through the rest of my modules to make sure this isn't in there as it isn't related like it says.

Comment (dreispt) [reply project_tag/README.rst:80]: No problem.

File: project_tag/__manifest__.py
Comment (emagdalenaC2i) [inline project_tag/__manifest__.py]: Delete extra "

File: project_tag/__manifest__.py
Comment (emagdalenaC2i) [inline project_tag/__manifest__.py]: delete blank lines

File: project_tag/__manifest__.py
Comment (emagdalenaC2i) [inline project_tag/__manifest__.py]: delete comments

**Comment by emagdalenaC2i** [comment]
There are 2 files
project_tag/models/__pycache__/__init__.cpython-36.pyc
project_tag/models/__pycache__/project.cpython-36.pyc
that I think are not part of the module, maybe created with your editor?

**Review by dreispt — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@dreispt could you check my previous comment
"There are 2 files
project_tag/models/pycache/init.cpython-36.pyc
project_tag/models/pycache/project.cpython-36.pyc
that I think are not part of the module, maybe created with your editor?"

**Comment by patrickrwilson** [comment]
Is there anything else needed on this module or is it just waiting on additional approvals?

**Review by rafaelbn — APPROVED** [review]
Just functional review :+1: 

Remember @patrickrwilson a great rule proposed in OCA is "Review 3 PR for each 1 you make" :smiley:  then OCA rocks!

**Comment by rafaelbn** [comment]
Please answer https://github.com/OCA/project/pull/505#issuecomment-486141284 comment from @emagdalenaC2i  thanks!

**Comment by patrickrwilson** [comment]
@rafaelbn I like that rule, i've learned a lot in the last week from my pr's and I think i'm much more confident and ready to start reviewing so I will be.

@emagdalenaC2i @rafaelbn although my commits don't include the pycache folder, it's still showing for some reason and I don't know why. Do you know how I can go about removing that folder?

**Comment by rafaelbn** [comment]
@dreispt could you please help here?, your wisdom is very appreciated :smile:

**Comment by patrickrwilson** [comment]
I think i properly removed it by removing the files and folder from github UI. I've squashed the commits after doing that and it looks normal now.

File: project_tag/models/project.py
Comment (alexey-pelykh) [inline project_tag/models/project.py:4]: Copyright, license are missing

File: project_tag/models/__init__.py
Comment (alexey-pelykh) [inline project_tag/models/__init__.py:3]: license is missing

File: project_tag/__init__.py
Comment (alexey-pelykh) [inline project_tag/__init__.py:3]: license is missing

File: project_tag/views/project.xml
Comment (alexey-pelykh) [inline project_tag/views/project.xml]: `<field name="name" position="after">` would be enough

File: project_tag/views/project.xml
Comment (alexey-pelykh) [inline project_tag/views/project.xml]: Just use <field>

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: project_tag/__init__.py
Comment (alexey-pelykh) [inline project_tag/__init__.py]: As per module template, https://github.com/OCA/maintainer-tools/tree/master/template/module, `__init__` files should contain only license

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Please squash recent commits into one, and all good :)

**Comment by patrickrwilson** [comment]
@alexey-pelykh This has been squashed. Thanks

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Pull request merged**
